gravitl/netclient
# Gra 988/add host to network

-> add a host to network, host update logic

Testing steps: 
 **Can only be tested by running API's manually , UI changes are yet to be done**
- [x] On adding the host to the network check if the interface is configured - check whether network routes are added to the interface
     `ifconfig utun69 or ifconfig netmaker` to verify the routes and vice-versa for removing the host from the network

- [x]  Check if peers of the network are added/removed 
- [x] verify peer's connections are working



** Removing the host from the network is done via delete node logic as is
